packages/trader/src/Modules/Trading/Helpers/video-config.ts

```diff
@@ -0,0 +1,24 @@
+import { TVideo } from '../Components/Form/ContractType/types';
+
+export const getDescriptionVideoId = (videos: TVideo[] = [], trade_type = '') => {
+    // finds a video id for the exact trade type by the video file name
+    return videos.find(video => video.meta.name.startsWith(`dtrader_${trade_type}`))?.uid ?? '';
+};
+
+export const getDescriptionVideos = async (is_dark_mode_on = false): Promise<TVideo[]> => {
+    const endpoint = `https://api.cloudflare.com/client/v4/accounts/${
+        process.env.CLOUDFLARE_ACCOUNT_ID
```

Comment:
videos in the test link will start working only after these env vars will be added to Vercel.